documentation/source/user_section/tutorials/lesion-analysis.rst

```diff
@@ -0,0 +1,16 @@
+.. _lesion-analysis:
+
+Lesion analysis
+###############
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/lesion-analysis_overview.png
```

Comment:
I don't see well white police over orange background
Maybe consider changing the font, since times new roman is never used in the other tutorials

Reply:
Good point! I change fonts to Arial and yellow to black at https://github.com/spinalcordtoolbox/doc-figures/commit/ca2dc0a0634cc68871abf97be2e574b002645bf4. What do you think?